tests/tc-001-val-large-txs.go

```diff
@@ -0,0 +1,23 @@
+package tests
+
+import (
+	appsync "github.com/celestiaorg/test-infra/tests/app-sync"
+	"github.com/testground/sdk-go/run"
+	"github.com/testground/sdk-go/runtime"
+)
+
+// Test-Case #001 - Validators submit large txs
+// Description is in docs/test-plans/001-Big-Blocks/test-cases
+func ValSubmitLargeTxs(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
```

Comment:
`func ValSubmitLargeTxs(runenv *runtime.RunEnv, initCtx *run.InitContext) (err error)`

So you can avoid creating a local variable at L12

Reply:
True. Thanks for the review. Done